dspace-api/src/main/java/org/dspace/embargo/EmbargoServiceImpl.java

```diff
@@ -142,9 +142,9 @@ public DCDate getEmbargoTermsAsDate(Context context, Item item)
         // sanity check: do not allow an embargo lift date in the past.
         if (liftDate.before(new Date()))
         {
-            throw new IllegalArgumentException(
-                    "Embargo lift date must be in the future, but this is in the past: "
-                            + result.toString());
+            log.info("Embargo lift date must be in the future, but this is in the past");
+            //DS-3321, existing code assumes a null lift date when in the past
```

Comment:
A few clarifications to my questions above..

To be fair, I'm not against the changes suggested in the PR.  It's just that this is throwing up potential red flags in my mind... and I want to be certain we have our reasons right...currently it seems to be a "this worked, so maybe it's right" sorta PR.  So, I'm _not_ a -1.  But, I am currently a 0 vote, and I'd need further convincing to move to a +1 vote.